kahuna/public/js/components/gr-image-metadata/gr-image-metadata.js

```diff
@@ -40,6 +40,28 @@ module.controller('grImageMetadataCtrl', [
     ctrl.metadata = ctrl.image.data.metadata;
     ctrl.identifiers = ctrl.image.data.identifiers;
 
+    mediaApi.fileMetadata(ctrl.image).then(resource => {
```

Comment:
We probably don't want to read `fileMetadata` as a matter of course. This can be in excess of 1Mb in many cases and we'll only ever want a tiny fraction of the content. Either the field aliases should be automatically added to the main image endpoint or a new endpoint that only serves those values can be created.